feature/session/src/main/java/io/github/droidkaigi/confsched2020/session/ui/SessionsFragment.kt

```diff
@@ -179,6 +181,22 @@ class SessionsFragment : DaggerFragment() {
             ) { checked, level ->
                 sessionsViewModel.filterChanged(level, checked)
             }
+
+            if (Build.VERSION.SDK_INT == Build.VERSION_CODES.LOLLIPOP) {
```

Comment:
For `LOLLIPOP_MR1`, I don't know that this workaround is needed or not.
I hope that someone verify this workaround on Lollipop MR1 device.

Reply:
I applied your fix to a branch I'm working on to fix #122  -- It worked flawlessly, however, your fix #118 wasn't quite consistent. Upon the first launch, it works but when you press say the back button and launch the app, the filters and the day 1 list are not shown. Though the favorite icon keeps work. See attached gif

![fix-layout-issue](https://user-images.githubusercontent.com/73175/72668478-a8e5f600-3a27-11ea-82dc-7def23c3d8e7.gif)

Reply:
@eyedol Thank you for reporting and sharing. And sorry for late replying.
Currently, I have no idea to solve that problem about the process of restoring views. Because I don't understand what is happening at restoring.

@takahirom Have you created the issue reported at above comment?